CLAUDE.md

```
@@ -98,6 +98,48 @@ Moya + 커스텀 `NetworkClient(actor)` 조합으로 구성됩니다.
+| design | UI 수정 |
+| typo | 오타 수정 |
+| mod | 폴더 구조 이동 및 파일 이름 수정 |
+| add | 파일 추가 |
+| del | 파일 삭제 |
+| refactor | 코드 리팩토링 |
+| init | 프로젝트 세팅 |
+| chore | 배포, 빌드 등 기타 작업 |
+| merge | 브랜치 병합 |
+
+### Example
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)